tests/dsl/ptg/user-defined-functions/udf.jdf

```diff
@@ -79,14 +93,32 @@ UD_HASH_STRUCT(m, n) [ hash_struct = ud_hash_struct ]
 
 READ X <- A(m, n)
 
-BODY
+BODY [ evaluate = never_here
```

Comment:
I like the example in general but we do not have an example of 'prefer_here' or 'here_if_load_lower_than', that would be also a common case that would be good to showcase.

Reply:
This test checks the `evaluate` support in the infrastructure, they are not meant as an showcase for users. That being said, I don't think I would know how to do anything related to the load without access to internal information.